unified-runtime/scripts/core/exp-bindless-images.yml

```diff
@@ -925,6 +927,25 @@ returns:
         - "`pMem == NULL`"
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Checks whether the device supports importing the specified external memory handle type"
+class: $xBindlessImages
+name: SupportsImportingHandleTypeExp
```

Comment:
a more conventional way to do this would be to add a corresponding capabilities bitfield for handle types and have it query-able via urDeviceGetInfo, like how kernel launch property support is reported https://oneapi-src.github.io/unified-runtime/core/api.html#ur-kernel-launch-properties-flags-t